apps/dotcms-ui/src/app/view/components/_common/dot-site-selector/dot-site-selector.component.ts

```diff
@@ -222,4 +226,15 @@ export class DotSiteSelectorComponent implements OnInit, OnChanges, OnDestroy {
         this.sitesCurrentPage = [...items];
         this.updateCurrentSite(this.siteService.currentSite);
     }
+
+    private setSelectedSite(): void {
```

Comment:
Fix: When open the theme selector, the selected site and themes should be the selected by the user and not the current site. 

When the siteSelector receive and id in the `@input` that will be the one selected instead the current site.